platforms/hyperledger-indy/configuration/roles/create/k8_component/vars/main.yaml

```diff
@@ -0,0 +1,4 @@
+k8_templates:
+  namespace: namespace_component.tpl
+  aws-baremetal-storageclass: eks_storageclass.tpl
```

Comment:
aws-baremetal maps to EKS? I would go for aws-storageclass : eks_storageclass.tpl and aws-baremetal-storageclass: aws_storageclass.tpl